everett/manager.py

```diff
@@ -46,6 +48,30 @@ def parse_bool(val):
     raise ValueError('%s is not a valid bool value' % val)
 
 
+def parse_env_file(envfile):
+    """Parse the content of an iterable of lines as .env
+
+    Return a dict of config variables.
+
+    >>> parse_env_file(['DUDE=Abides'])
+    {'DUDE': 'Abides'}
+
+    """
+    data = {}
+    for line in envfile:
+        line = line.strip()
+        if not line or line.startswith('#') or '=' not in line:
```

Comment:
Honestly this is mostly lifted from `decouple`. I agree though that if it's a line that's not a comment, blank, or valid entry it should probably be an exception. I'll update.